src/components/Animator/components/CSSTransitionWrapper.js

```diff
@@ -26,7 +26,8 @@ class CSSTransitionWrapper extends React.Component {
     };
   }
 
-  componentWillReceiveProps(props) {
+  // eslint-disable-next-line camelcase
+  UNSAFE_componentWillReceiveProps(props) {
```

Comment:
@mykas 
This change is what broke us the last time, no?
It breaks the build if I don't add the prefix so I'm not sure what to do in this case.